Source/Scene/ShadowMap.js

```diff
@@ -185,8 +185,9 @@ define([
         this._needsUpdate = true;
 
         // In IE11 polygon offset is not functional.
+        // TODO : Also disabled for Chrome temporarily. Re-enable once https://groups.google.com/forum/#!topic/webgl-dev-list/E1dAG65QBhg is resolved.
```

Comment:
If this is specifically an ANGLE problem, why are we disabling on Chrome in all cases?  We have lots of non-ANGLE Chrome users (Mac/Linux/Android/iOS)